### ISSUE TITLE
Rename extension markdown files to README

### ISSUE THREAD

**Issue by jridgewell** [ampproject/amphtml#1304]
Eg. [amp-access](https://github.com/ampproject/amphtml/tree/master/extensions/amp-access) should have its markdown file renamed to `README.md`. Github displays these files [specially](https://github.com/ampproject/amphtml/tree/master/extensions#readme), so they do not need to be clicked on when the viewing the extension directory.

**Comment by cramforce** [comment]
Good point. This may require some changes on the docs side.
CC @pbakaus @Meggin

**Comment by Meggin** [comment]
Good idea. Just need to coordinate with ampproject.org pulls to make sure we still get the reference in the site. I can take a look.

**Comment by britice** [comment]
I'd like to take this on as a warm-up for future amp-analytics contributions.

**Comment by Meggin** [comment]
Brian, when you say take this on, will you be taking on the script that
pulls in the latest reference docs into the /docs github repo, or just
changing in amphtml repo?

It's important we don't break the scripts, so please keep me and Paul
Bakaus in the loop.

**Comment by britice** [comment]
Meggin,
> ..., will you be taking on the script that pulls in the latest reference docs into the /docs github repo, or just changing in amphtml repo?

I'll look into doing both.

> It's important we don't break the scripts, so please keep me and Paul Bakaus in the loop.

That make sense. I'll keep you both updated on my progress. If I run into a difficulty which leads me to give up on this and move to an easier starter task, I'll be sure to post something here.

**Comment by Meggin** [comment]
Awesome. Thanks!

**Comment by britice** [comment]
My branch for this issue: https://github.com/britice/amphtml/tree/readme-renames

That should cover the changes for this repo but I'll look into what needs to be done in the reference docs script before I create a PR.

**Comment by britice** [comment]
@Meggin @pbakaus, I've added an issue in the docs repo (https://github.com/ampproject/docs/issues/20) concerning the script to pull reference docs.

What's your opinion on the order the changes should be made. In it's current form, the changes I made to the docs script will use either `README.md` or the current format (e.g. `amp-font.md`). This would allow for backwards compatibility until all the READMEs are renamed with a PR in this repo.

If backwards compatibility is not a concern, the import script can likely be simplified.
